[Game/AI] Improved TalkBehavior functionality:

NL0bP opened the pull request:
- Data is now pulled from npc_chat_bubbles and ai_events tables;

[Game/AI] Enhanced IdleBehavior and HoldPositionBehavior:
- Added transition to OnTalk with a 5-minute cooldown per character;

[Packets] Fixed SCChatBubblePacket and added SCPlaySoundPacket;

NL0bP committed:
[Game/AI] Improved TalkBehavior functionality:
- Data is now pulled from npc_chat_bubbles and ai_events tables;

[Game/AI] Enhanced IdleBehavior and HoldPositionBehavior:
- Added transition to OnTalk with a 5-minute cooldown per character;

[Packets] Fixed SCChatBubblePacket and added SCPlaySoundPacket;

hermanocabral commented on AAEmu.Game/GameData/AiGameData.cs:
```diff
@@ -144,6 +146,81 @@ public void Load(SqliteConnection connection)
+            _npcChatBubbles[bubble.AiEventId] = bubble;
+        }
+
+        Logger.Info($"Загружено {_npcChatBubbles.Count} записей из npc_chat_bubbles.");
```
Не все говорят по-русски 😅

hermanocabral replied:
"Loaded {_npcChatBubbles.Count} records from npc_chat_bubbles."?

NL0bP replied:
> Не все говорят по-русски 😅

I fixed the text.

hermanocabral replied:
I know, I had started the review 1hr ago but had to leave for a meeting. Just got back now hahaha

hermanocabral commented on AAEmu.Game/GameData/AiGameData.cs:
```diff
@@ -144,6 +146,81 @@ public void Load(SqliteConnection connection)
+            _aiEventsByNpc[aiEvent.NpcId].Add(aiEvent);
+        }
+
+        Logger.Info($"Загружено {_aiEventsByNpc.Count} записей из ai_events.");
```
"Loaded {_aiEventsByNpc.Count} records from ai_events."?

NL0bP committed:
Translated the Russian text into English.

hermanocabral reviewed (approved):
LGTM

NL0bP committed:
Fixed spyglass spawn: now requires ParentWorld;
Removed all spyglasses: they now spawn only when interacting with the stationary lookout.
Fixed name display for Npcs and Doodads.

ZeromusXYZ reviewed (dismissed):
Looks like this shouldn't conflict with the `ChatBubble` effect 👍

NL0bP committed:
Fixed errors in the .json spawn file.
Fixed name display for NPCs and objects (Doodads) again.
Fixed IdleBehavior.

hermanocabral reviewed (approved):
LGTM
